WoW-Realm-Tracker/Models/Realm.swift

```diff
@@ -1,27 +1,36 @@
 import Runes
 import Argo
+import Curry
 
 struct Realm {
-    let name: String
-    let type: String
-    let status: Bool
+    enum RealmType: String, CustomStringConvertible {
+        case PvP = "pvp"
```

Comment:
It is just an API conversion concern so i like your idea about having that just happen at parse time rather than exist at all times.

I tried the above but I'm getting this error:

> /Volumes/Data/Users/dgalarza/Projects/MobileRealmStatus/WoW-Realm-Tracker/Models/Realm.swift:37:13: Binary operator '<*>' cannot be applied to operands of type 'Decoded<(Bool) -> Realm>' (aka 'Decoded<Bool -> Realm>') and 'Decoded<String>'

I'm not quite sure what its getting at.